test/doctool/test-local-md-file-reference-regex.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
What's the command you ran to make this test run?

Reply:
`make test-doc`

https://github.com/nodejs/node/blob/73900983f0180f563b9a6959fee378789db6854c/Makefile#L606